go/vt/vtorc/config/config.go

```diff
@@ -82,6 +83,7 @@ func RegisterFlags(fs *pflag.FlagSet) {
 	fs.DurationVar(&topoInformationRefreshDuration, "topo-information-refresh-duration", topoInformationRefreshDuration, "Timer duration on which VTOrc refreshes the keyspace and vttablet records from the topology server")
 	fs.DurationVar(&recoveryPollDuration, "recovery-poll-duration", recoveryPollDuration, "Timer duration on which VTOrc polls its database to run a recovery")
 	fs.BoolVar(&ersEnabled, "allow-emergency-reparent", ersEnabled, "Whether VTOrc should be allowed to run emergency reparent operation when it detects a dead primary")
+	fs.BoolVar(&convertTabletWithErrantGTIDs, "convert-tablets-with-errant-gtids", convertTabletWithErrantGTIDs, "Whether VTOrc should be changing the type of the tablets with errant GTIDs to DRAINED")
```

Comment:
This variable doesn't seem to be actually used? The function `ConvertTabletWithErrantGTIDs()` just returns `ersUsed`.

Reply:
Fixed.